catalog/spatial/geowebcache/geowebcache-app/pom.xml

```diff
@@ -26,7 +26,7 @@
     <name>DDF :: Spatial :: GeoWebCache App</name>
 
     <properties>
-        <gwc-server.version>0.6</gwc-server.version>
+        <gwc-server.version>0.7.0-SNAPSHOT</gwc-server.version>
```

Comment:
Good point. Fixed. We just can't merge this until the geo web cache stuff is released.